src/genegraph/util/fs.clj

```diff
@@ -0,0 +1,13 @@
+(ns genegraph.util.fs
+  (:require [clojure.java.io :as io])
+  (:import [java.io File]))
+
+(defn ensure-target-directory-exists!
+  "Create directory at PATH if it does not already exist.
+  Return false if directory cannot be created or already
+  exists as something other than a directory."
+  [path]
+  (let [dir (io/file path)]
+    (if (and (.exists dir) (.isDirectory dir))
+      true
+      (.mkdir dir))))
```

Comment:
Try `clojure.java.io/make-parents` instead.

Reply:
Good suggestion; I think that doesn't quite do the same thing at this, but one could phase this out by calling make-parents prior to opening the file in question, I think.